taxonomy/models.py

```diff
@@ -50,8 +50,7 @@ class Meta:
     @transaction.atomic
     def save(self, *args, **kwargs):
         """Handle slugs."""
-        if self.id is None or self.name != get_object_or_404(
-                Vocabulary, id=self.id).name:
+        if self.id is None:
```

Comment:
it i allow this code ``` self.name != get_object_or_404(Vocabulary, id=self.id).name``` each time user updates a vocabulary it updates it slug and create new entry in db.